automatic/freecad/update_helper.ps1

```diff
@@ -23,17 +22,8 @@ param(
       $veri = ((($url64 -split('\/'))[-1]) -replace( "(x\d{2})|(_\d{2}\-py\d{2})|(\-)?([A-z])+?(\-)|(\.$ext)", ''))
       "veri -$veri-" | Write-Warning
       $DevRevision,$year,$month,$day = (($veri -replace('\-','.') ) -split('\.'))
-      [version]$DevVersion = (( Get-Content "$ScriptLocation\freecad.json" | ConvertFrom-Json ).dev) -replace("-$kind",'')
-      # Catch to make sure version will be the latest version
-      if (( $DevVersion.Minor -lt ($year.Substring(0,2)) ) -and ( $DevVersion.Build -gt $filler )) { [version]$DevVersion = "0.20.${filler}" }
-      if (($DevRevision -match "\d{5}") -and ($DevRevision -le "29192")){
-        "Revision is less than it was on date 6/20/2022 which could mean a new version minor" | Write-Warning
-        "Going to increase the version minor by 1" | Write-Warning
-        [version]$version = ( ( ($DevVersion.Major),(($DevVersion.Minor + 1)),($DevVersion.Build),($DevRevision) ) -join "." )
-      } else {
-        "Standard Versioning for $DevRevision dated ${month}-${day}-${year}" | Write-Warning
-        [version]$version = ( ( ($DevVersion.Major),($DevVersion.Minor),($DevVersion.Build),($DevRevision) ) -join "." )
-      }
+      "Standard Devlopment Versioning for $DevRevision dated ${month}-${day}-${year}" | Write-Warning
```

Comment:
Small typo, which just annoyed me (would take care of it, but unfortunately do not have the time right now).

```suggestion
      "Standard Development Versioning for $DevRevision dated ${month}-${day}-${year}" | Write-Warning
```

Reply:
I'll fix it